wrappers/src/error_handling.hpp

```diff
@@ -92,16 +92,16 @@ class NotNullableException : public std::runtime_error {
 class PropertyTypeMismatchException : public std::runtime_error {
 public:
     PropertyTypeMismatchException(std::string object_type, std::string property, std::string property_type, std::string actual_type)
-        : std::runtime_error(util::format("Property type mismatch: %1.%2 is of type %3, but the value is of type %4")) {}
+        : std::runtime_error(util::format("Property type mismatch: %1. %2 is of type %3, but the supplied value is of type %4", object_type, property, property_type, actual_type)) {}
```

Comment:
There shouldn't be a space between `%1` and `%2`. The idea here is to have a message like `Foo.Bar is of type string, but the supplied value is of type int`. With the space, it looks like end of a sentence after `Foo`.

Reply:
In my mind that was a real full stop. Like `Mistmatch: Foo. "Property" Bar (clearly of Foo) is of type....`
But now that you say that, it just makes a lot more sense. 😅